lib/passenger.rb

```diff
@@ -0,0 +1,32 @@
+class Passenger
+  attr_reader :name, :age
+  def initialize(passenger_details)
+    @name = passenger_details["name"]
+    @age = passenger_details["age"]
+    @driver = false
+  end
+
+  def adult?
+    if @age >= 18
+      true
+    else
+      false
+    end
+  end
+
+  def driver?
+    if @driver == true
+      true
+    else
+      false
+    end
+  end
+
+  def drive
+    if @age >= 18
+      @driver = true
+    else @age < 18
+      @driver = false
+    end
```

Comment:
```suggestion
    if adult?
      @driver = true
    end
```

Ruby won't evaluate anything to the right of `else`. `else` is a catch-all meaning there are no conditions to be considered. `else` happens when all the `if` and `elsif`s did not match and we still want to perform an action. In this case, we ONLY want to perform an action if the person is an adult so we can truncate this to just be the `if` portion of this `if` statement. We can also use the `adult?` method that you wrote on line 26 so that you don't need to re-write this logic a second time.